src/page.js

```diff
@@ -65,11 +65,6 @@ class ResolvedPage extends PendingPage {
   get isPending() { return false; }
   get isResolved() { return true; }
   get isSettled() { return true; }
-
-  // Redundant, since we can call resolvedPage.resolve(resolvedPage.unfilteredRecords, resolvedPage.filterCallback) to reapply filters
-  filter(){
-    return new ResolvedPage(this, this.unfilteredData, this.filterCallback);
```

Comment:
Why call page.filter() when we can just resolve the page with out current attributes?
https://github.com/flexyford/impagination/pull/28/files#diff-6999e25e70781d24e5c8cd2c8d4b4150R205